build.gradle.kts

```diff
@@ -19,6 +22,11 @@ plugins.withType<JavaPlugin>().configureEach {
 group = "org.junit-pioneer"
 description = "JUnit 5 Extension Pack"
 
+// used in pom and bintray
+val issueTracker = "https://github.com/junit-pioneer/junit-pioneer/issues"
+val scm = "https://github.com/junit-pioneer/junit-pioneer.git"
+val website = "https://junit-pioneer.org/"
```

Comment:
would it make sense to put those properties into the gradle.properties? smae for group and description. To somehow separate the build logic and the hard coded values?

Reply:
`group` and `description` is gradle Project properties, as I see we must set those in `build.gradle.kts`

next I see - when we define something in `gradle.properties` we must refer it in `build.gradle.kts` like 
```
val junitVersion : String by project`
```
so maybe another proposition ...

Reply:
yeah i am still not used to the kotlin gradle setup :D i am more aware of the gradle approach.
Still it would be nice to have the actual data defined outside the building blocks, as it is imho easier to maintain. but i might be the only one with this thinking :D - so for me it is fine, not sure what the rest thinks